spec/index.bs

```diff
@@ -6034,12 +6066,20 @@ type and each [=binding type=] has an associated [=internal usage=], given by th
                 : |entry|.{{GPUBindGroupLayoutEntry/buffer}}?.{{GPUBufferBindingLayout/type}}
                     is {{GPUBufferBindingType/"storage"}} or {{GPUBufferBindingType/"read-only-storage"}}
                 :: Consider 1 {{supported limits/maxStorageBuffersPerShaderStage}} slot to be used.
```

Comment:
IIUC because of the rules about adjusting the limits, we could actually apply this only to COMPUTE. It would be a bit clearer to implementers, though more confusing overall so we would probably want a `Note:` explaining why.

Reply:
Done